moped-editor/src/layouts/DashboardLayout/NavBar/NavigationSearchQueryConf.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * The Query configuration (now also including filters)
+ * @constant
+ * @type {object}
+ * @augments NavigationSearchFiltersConf
+ * @default
+ */
+export const NavigationSearchQueryConf = {
+  options: {
+    useQuery: {
+      fetchPolicy: "cache-first", // Default is "cache-first", or use "no-cache"
```

Comment:
totally irrelevant, but for whatever reason the cache policy does not seem to work for queries generated  by the abstract. we have https://github.com/cityofaustin/atd-data-tech/issues/10799 to address that.